rsocket/internal/InMemResumeManager.cpp

```diff
@@ -178,19 +190,46 @@ void InMemResumeManager::sendFramesFromPosition(
 void InMemResumeManager::onStreamClosed(StreamId streamId) {
   // This is crude. We could try to preserve the stream type in
   // RSocketStateMachine and pass it down explicitly here.
-  activeRequestStreams_.erase(streamId);
-  activeRequestChannels_.erase(streamId);
-  activeRequestResponses_.erase(streamId);
+  requesterRequestStreams_.erase(streamId);
```

Comment:
Right.  This is just a sample implementation, so I did not both bother optimizing the existing code much.  Eventually, `InMemResumeManager` will not care about any of this.  Any other implementation of ResumeManager should have a optimized way of doing this (and I will look at changing the API to facilitate that)